defaults/main.yml

```diff
@@ -52,8 +52,8 @@ jenkins2_local_cert_file_name: '{{ ansible_hostname }}.ca-cert.pem'
 # SSH_KEYS generation properties
 jenkins2_ssh_keys_generate: false
 jenkins2_ssh_keys_private_keyname: "id_rsa"
-jenkins2_ssh_keys_size: 4096
-jenkins2_ssh_keys_inv_slave_groupname: "jenkins_ssh_slaves"
+jenkins2_ssh_keys_size: 2048
+jenkins2_ssh_slave_key_fetch: '/tmp/slave.pub'
 
 jenkins2_ssh_keys_slave_group: "{{ groups[jenkins2_ssh_keys_inv_slave_groupname]|default([''])}}"
```

Comment:
jenkins2_ssh_keys_inv_slave_groupname was removed from readme and defaults

Reply:
Fixed